command/node_status.go

```diff
@@ -632,9 +638,18 @@ func (c *NodeStatusCommand) formatNode(client *api.Client, node *api.Node) int {
 }
 
 func (c *NodeStatusCommand) outputAllocInfo(node *api.Node, nodeAllocs []*api.Allocation) error {
+	var nodeMaxAllocs string
+	if node.NodeMaxAllocs == nil {
+		nodeMaxAllocs = "N/A"
+	} else {
+		nodeMaxAllocs = strconv.Itoa(node.NodeMaxAllocs.MaxAllocs)
+	}
 	c.Ui.Output(c.Colorize().Color("\n[bold]Allocations[reset]"))
 	c.Ui.Output(formatAllocList(nodeAllocs, c.verbose, c.length))
 
+	c.Ui.Output(c.Colorize().Color("\n[bold]Node Max Allocations[reset]"))
+	c.Ui.Output(nodeMaxAllocs)
```

Comment:
This format is how we present a table (like the allocation list), so it's out of place to have a single value here. Maybe we could lift this up into the `Allocated Resources` section? 

For example, we could go from this:

```
Allocated Resources
CPU            Memory          Disk
100/25400 MHz  100 MiB/31 GiB  300 MiB/6.2 GiB
```

to something like this:

```
Allocated Resources
CPU            Memory          Disk             Alloc Count
100/25400 MHz  100 MiB/31 GiB  300 MiB/6.2 GiB  1/20
```

Reply:
This is not only cleaner but allowed me to delete like 5 lines and 2 files from the PR!